templates/managed/kafka.html

```diff
@@ -154,151 +163,182 @@ <h3>Cloud architecture</h3>
       </tr>
       <tr>
         <td><strong>On-premise</strong></td>
-        <td colspan="2">Any OpenStack, bare metal and key third-party providers such as VMWare.</td>
-      </tr>
-      <tr>
-        <td><strong>Hybrid-cloud</strong></td>
-        <td colspan="2">Persistent streaming between all cloud types and management across all</td>
-      </tr>
-      <tr>
-        <td><strong>Multi-cloud</strong></td>
-        <td colspan="2">Persistent streaming between all cloud types and management across all</td>
-      </tr>
-    </table>
+        <td colspan="2">Any OpenStack, bare metal and key third-party providers such as VMWare.<br />
+          We can support your specific cloud provider - <a href="/managed/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>.</td>
```

Comment:
I don't think this one should be a button. Would be more consistent as a link.

Reply:
you are right, moving too fast!